### PR TITLE
Button ripple feature + sass-lint on build + Github templates ...

### LINKED ISSUE #126: Import/export multiple components
Fixing support issue to use:

E.g.
module/components/AppBar/index.jsx
```
export { default as AppBar } from './AppBar';
export { default as AppBarItem } from './AppBarItem';
```
module/index.js
```
export * from './components/AppBar';
```

### LINKED ISSUE #127: Adding size property to Avatar
Add prop size to Avatar component and make the default size to 50px

### PR DESCRIPTION
**Pull request by jeradrutnam**
Updated button ripple animation along with some fixes [#94]
Added Style-lint to DEV Mode [#122]
Added PULL_REQUEST_TEMPLATE.md [#123]
Added ISSUE_TEMPLATE.md [#124]
Fixed stylelint errors [#125]
[Fixed #126] Import/export multiple components
[Fixed #127] Adding size property to Avatar

### CODE REVIEW

**Review by dakshika — CHANGES_REQUESTED** [review]
Use standard templete

File: module/stylesheets/components/_avatar.scss [hunk 1]
```diff
@@ -1,26 +1,27 @@
 .avatar {
   display: inline-flex;
-  height: 50px;
   width: 50px;
+  height: 50px;
```
Comment (dakshika) [inline module/stylesheets/components/_avatar.scss]: Isn't this a prop?

**Review by dakshika — CHANGES_REQUESTED** [review]

File: module/stylesheets/components/_avatar.scss (hunk 1 above)
Comment (jeradrutnam) [reply module/stylesheets/components/_avatar.scss]: Yes it has to be. Didn't do any feature changes to them. All I did was fixing the linting errors and and some other build issues. Yes we need to update this component. I will fix it.

Comment (jeradrutnam) [reply module/stylesheets/components/_avatar.scss]: Done

**Review by dakshika — APPROVED** [review]

**Pull request merged**